install/installer/pkg/config/v1/config.go

```diff
@@ -157,6 +157,8 @@ type ObjectStorage struct {
 type ObjectStorageS3 struct {
 	Endpoint    string    `json:"endpoint" validate:"required"`
 	Credentials ObjectRef `json:"credentials" validate:"required"`
+
+	BucketName string `json:"bucket"`
```

Comment:
NIT: Would you mind adding a comment to this config value that explains what happens when this bucket name is set (and more importantly what happens when this value is not set?

Reply:
done